mne/viz/misc.py

```diff
@@ -245,30 +245,8 @@ def plot_source_spectrogram(stcs, freq_bins, tmin=None, tmax=None,
 
 
 def _plot_mri_contours(mri_fname, surfaces, src, orientation='coronal',
-                       slices=None, show=True):
-    """Plot BEM contours on anatomical slices.
-
-    Parameters
-    ----------
-    mri_fname : str
-        The name of the file containing anatomical data.
-    surfaces : list of (str, str) tuples
-        A list containing the BEM surfaces to plot as (filename, color) tuples.
-        Colors should be matplotlib-compatible.
-    src : None | SourceSpaces
-        SourceSpaces object for plotting individual sources.
-    orientation : str
-        'coronal' or 'axial' or 'sagittal'
-    slices : list of int
-        Slice indices.
-    show : bool
-        Show figure if True.
-
-    Returns
-    -------
-    fig : Instance of matplotlib.figure.Figure
-        The figure.
-    """
```

Comment:
you can revert this change

Reply:
I'd rather leave them removed, I don't think they add anything beyond what the public documentation has